CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master / unreleased
 
+## 1.6.0-rc0 in progress
```

Comment:
Should be1.6.0-rc.0 (not the dot between rc and 0)

Reply:
👍